app/assets/javascripts/Components/repo_browser.jsx

```diff
@@ -168,15 +191,26 @@ class ManualCollectionForm extends React.Component {
               {I18n.t("submissions.collect.apply_late_penalty")}
             </label>
           </p>
-          <button
-            type="submit"
-            name="commit"
-            onClick={event => {
-              if (!confirm(I18n.t("submissions.collect.overwrite_warning"))) {
-                event.preventDefault();
-              }
-            }}
-          >
+          <p className="inline-labels">
+            <input
+              type="checkbox"
+              name="retain_existing_grading"
+              data-testid="chk_retain_existing_grading"
+              checked={this.state.retainExistingGrading}
+              hidden={!this.props.collected_revision_id}
+              disabled={!this.props.collected_revision_id} // prevent from sending info on submit
+              onChange={e => {
+                this.setState({retainExistingGrading: e.target.checked});
+              }}
+            />
+            <label
```

Comment:
Use `htmlFor` attribute to link this label to the corresponding `input`; make sure to do this in the modal as well.

FYI, you can test this in the UI by *clicking on the label itself*: when properly linked, clicking the label should act the same as checking the checkbox itself.